File: Solutions/Hard/1526. Minimum Number of Increments on Subarrays to an Array/Solution.cs
public class Solution {
    public int MinNumberOperations(int[] target) {
        int numberOfPasses = 0;
        int prev = 0;
        foreach(int num in target){
            if(num > prev) numberOfPasses += (num - prev);
            prev = num;
        }
        return numberOfPasses;
    }
}

File: Solutions/Hard/1526. Minimum Number of Increments on Subarrays to an Array/Solution.py
class Solution:
    def minNumberOperations(self, target: List[int]) -> int:
        numOfPasses: int = 0
        prev: int = 0
        for num in target:
            if(prev < num):
                numOfPasses += (num - prev)
            prev = num
        return numOfPasses